qa/suites/deepsea/tier2/igw/cluster/roles.yaml

```diff
@@ -1,5 +1,4 @@
 roles:
-  - [client.salt_master, mon.a, mgr.x, osd.0]
-  - [mon.b, mgr.y, osd.0]
-  - [mon.c, mgr.z, osd.0]
-  - [igw.0, client.0]
+  - [client.salt_master, mon.a, mgr.x, osd.0, igw.0]
+  - [igw.1, client.0]
+
```

Comment:
please drop the concluding newline (probably harmless, but still)

Reply:
done